sdks/python/apache_beam/runners/job/utils.py

```diff
@@ -0,0 +1,32 @@
+#
```

Comment:
Should we generalize https://github.com/apache/beam/blob/master/sdks/python/apache_beam/internal/gcp/json_value.py ?